cmd/server/main.go

```diff
@@ -3,18 +3,23 @@ package main
 import (
 	"log"
 	"net"
+	"os"
 
 	"github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 	"github.com/mrunalp/ocid/server"
 	"google.golang.org/grpc"
 )
 
 const (
-	port = ":49999"
+	unixDomainSocket = "/var/run/ocid.sock"
 )
 
 func main() {
-	lis, err := net.Listen("tcp", port)
+	// Remove the socket if it already exists
+	if err := os.Remove(unixDomainSocket); err != nil {
```

Comment:
@mrunalp , should we first check whether `unixDomainSocket` exists before `os.Remove` it?